libbeat/_meta/fields.ecs.yml

```diff
@@ -1,5 +1,5 @@
 # WARNING! Do not edit this file directly, it was generated by the ECS project,
-# based on ECS version 1.4.0.
+# based on ECS version 1.5.0-dev.
```

Comment:
Don't forget to grab the version from https://github.com/elastic/ecs/pull/718 ;-)